QueryBuilderTests/QueryBuilderTests.swift

```diff
@@ -431,6 +431,15 @@ class OperationTests: XCTestCase {
         "}\n" +
     "}")
     }
+    
+    func testInitializersOnSelectionTypeArray() {
+        let fields: [Field] = ["scalar"]
+        let _ = Object(name: "object", selectionSet: fields)
+        let _ = QueryBuilder.Operation(type: .query, name: "Query", selectionSet: fields)
+        let _ = InlineFragment(typeName: "Derp", selectionSet: fields)
+        let frag = FragmentDefinition(name: "frag", type: "Fraggie", selectionSet: fields)
+        XCTAssertNotNil(frag)
```

Comment:
I added initializers to all 4 of those types so that `[SelectionType]` is an accepted parameter, failure case is the compiler gets mad.
This specific initializer can return `nil` so that's just an additional sanity check.